### ISSUE TITLE
Build fails with `@nuxt-themes/elements`

### ISSUE THREAD

**Issue by kokamkarsahil** [nuxt/content#2057]
### Environment

- Operating System: Linux
- Node Version:     v16.17.0
- Nuxt Version:     3.4.3
- Nitro Version:    2.3.3
- Package Manager:  yarn@1.22.19
- Builder:          vite
- User Config:      modules, extends, content
- Runtime Modules:  @nuxt/content@2.6.0
- Build Modules:    -

### Reproduction

Steps to reproduce:
1. Go to nuxt.new select content starter doc-driven template or use `npx nuxi init -t doc-driven demo`.
2. Setup `@nuxt-themes/elements` - https://elements.nuxt.space/.
3. Run build step.

---
Reproduction: https://github.com/kokamkarsahil/Nuxt-Reproducible-Bug 

### Describe the bug

Adding `@nuxt-themes/elements` causes build to fail with nuxt content.

### Additional context

[@nuxt-themes/elements](https://github.com/nuxt-themes/elements) is currently a private so opening issue here.

Thanks!

### Logs

```shell
ERROR  Unexpected token (Note that you need plugins to import files that are not JavaScript)  4:15:05 PM
file: demo/node_modules/@nuxt-themes/elements/components/globals/Alert.vue?vue&type=style&index=0&scoped=7c235730&lang.ts?inline&used:1:0
1: .alert{margin-top:var(--space-8);margin-bottom:var(--space-8);border-radius:var(--radii-md);padding-left:var(--space-...
   ^


 ERROR  Unexpected token (Note that you need plugins to import files that are not JavaScript)  4:15:05 PM

  at error (node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:2125:30)
  at Module.error (node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:13423:16)
  at Module.tryParse (node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:14130:25)
  at Module.setSource (node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:13732:39)
  at ModuleLoader.addModuleSource (node_modules/vite/node_modules/rollup/dist/es/shared/node-entry.js:23705:20)
```
```

**Comment by farnabaz** [comment]
This is related to the latest version of Nuxt, and it is fixed in https://github.com/nuxt/nuxt/pull/20679
The fix is not released yet, so you can either use Nuxt's edge version or downgrade until Nuxt 3.5 release.

Check this PR to keep track of Nuxt release https://github.com/nuxt/nuxt/pull/20586

**Comment by kokamkarsahil** [comment]
> This is related to the latest version of Nuxt, and it is fixed in [nuxt/nuxt#20679](https://github.com/nuxt/nuxt/pull/20679) The fix is not released yet, so you can either use Nuxt's edge version or downgrade until Nuxt 3.5 release.
> 
> Check this PR to keep track of Nuxt release [nuxt/nuxt#20586](https://github.com/nuxt/nuxt/pull/20586)

Got it, thanks a lot! Spend my time rewriting those components to work ╥_╥
